.github/workflows/promote.yml

```
@@ -23,105 +38,154 @@ jobs:
       N8NAC_POSTHOG_HOST: ${{ vars.N8NAC_POSTHOG_HOST }}
       N8NAC_TELEMETRY_ENV: prod
     outputs:
-      pending: ${{ steps.pending.outputs.pending }}
+      release_branch: ${{ steps.resolve.outputs.release_branch }}
+      stable_tag: ${{ steps.resolve.outputs.stable_tag }}
     steps:
-      - name: Checkout
+      - name: Resolve RC tag
+        id: resolve
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2129:style:7:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)